kube/services/jobs/fence-cleanup-expired-ga4gh-info-cronjob.yaml

```diff
@@ -0,0 +1,113 @@
+---
+apiVersion: batch/v1beta1
+kind: CronJob
+metadata:
+  name: cleanup-expired-ga4gh-information
+spec:
+  schedule: "*/5 * * * *"
+  successfulJobsHistoryLimit: 2
+  failedJobsHistoryLimit: 2
+  concurrencyPolicy: Forbid
+  jobTemplate:
+    spec:
+      backoffLimit: 0
+      template:
+        metadata:
+          labels:
+            app: gen3job
+        spec:
+          serviceAccountName: useryaml-job
+          volumes:
+            - name: yaml-merge
+              configMap:
+                name: "fence-yaml-merge"
+            - name: config-volume
+              secret:
+                secretName: "fence-config"
+            - name: fence-google-app-creds-secret-volume
+              secret:
+                secretName: "fence-google-app-creds-secret"
+            - name: fence-google-storage-creds-secret-volume
+              secret:
+                secretName: "fence-google-storage-creds-secret"
+            - name: shared-data
+              emptyDir: {}
+          initContainers:
+          - name: fence
+            GEN3_FENCE_IMAGE
+            imagePullPolicy: Always
+            env:
+              - name: PYTHONPATH
+                value: /var/www/fence
+              - name: FENCE_PUBLIC_CONFIG
+                valueFrom:
+                  configMapKeyRef:
+                    name: manifest-fence
+                    key: fence-config-public.yaml
+                    optional: true
+            volumeMounts:
+              - name: "config-volume"
+                readOnly: true
+                mountPath: "/var/www/fence/fence-config-secret.yaml"
+                subPath: fence-config.yaml
+              - name: "yaml-merge"
+                readOnly: true
+                mountPath: "/var/www/fence/yaml_merge.py"
+                subPath: yaml_merge.py
+              - name: "fence-google-app-creds-secret-volume"
+                readOnly: true
+                mountPath: "/var/www/fence/fence_google_app_creds_secret.json"
+                subPath: fence_google_app_creds_secret.json
+              - name: "fence-google-storage-creds-secret-volume"
+                readOnly: true
+                mountPath: "/var/www/fence/fence_google_storage_creds_secret.json"
+                subPath: fence_google_storage_creds_secret.json
```

Comment:
Are the Google secrets needed?

Reply:
maybe not, this is just removing entries from the fence db. But my default has always been to include all the fence cfg in the deployments. we can remove and try it out